### PR TITLE
feat: Add qldb.Ledgers resource

### LINKED ISSUE #626: Add Support for AWS::QLDB::Ledger

### PR DESCRIPTION
**Pull request by roneli**
Closes #626

### CODE REVIEW

File: client/client.go [hunk 1]
```diff
@@ -8,6 +8,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/service/qldb"
+
```
Comment (disq) [inline client/client.go:12]: rogue newline

File: resources/provider/migrations/postgres/24_v0.10.16.up.sql [hunk 2]
```diff
@@ -87,3 +87,65 @@ ALTER TABLE IF EXISTS aws_mq_broker_users
...
+     "stream_name" text,
+     "arn" text,
+     "creation_time" timestamp without time zone,
+     "error_cause" text,
+     "exclusive_end_time" timestamp without time zone,
+     "inclusive_start_time" timestamp without time zone,
+     CONSTRAINT aws_qldb_ledger_journal_kinesis_streams_pk PRIMARY KEY(cq_id),
+     UNIQUE(cq_id)
+    );
+CREATE INDEX ON aws_qldb_ledger_journal_kinesis_streams (ledger_cq_id);
```
Comment (disq) [inline resources/provider/migrations/postgres/24_v0.10.16.up.sql:130]: is this a new thing/case we need to add to the migrations generator?

File: resources/services/qldb/ledgers.go [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+						Description: "The user-defined name of the QLDB journal stream.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "arn",
+						Description: "The Amazon Resource Name (ARN) of the QLDB journal stream.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "creation_time",
```
Comment (disq) [inline resources/services/qldb/ledgers.go:145]: not `creation_date_and_time` like the parent resource? inconsistent

**Review by disq — APPROVED** [review]
LGTM with nits

File: resources/services/qldb/ledgers.go (hunk 3 above)
Comment (roneli) [reply resources/services/qldb/ledgers.go:148]: That is how its in the original struct

**Comment by roneli** [comment]
Closed for #702

**Pull request closed without merging**